src/Raven.Server/Documents/Handlers/PullReplicationHandler.cs

```diff
@@ -32,5 +42,62 @@ public async Task DefinePullReplication()
                     json[nameof(OngoingTask.TaskId)] = index;
                 }, statusCode: HttpStatusCode.Created);
         }
+
+        [RavenAction("/databases/*/admin/certificates/feature", "PUT", AuthorizationStatus.Operator)]
+        public Task PutFeatureCertificateOnEdge()
+        {
+            return PutCertificate(PullReplicationDefinition.GetPrefix(Database.Name), includePrivateKey: true);
+        }
+
+        [RavenAction("/databases/*/info/remote-task/tcp", "GET", AuthorizationStatus.RestrictedAccess)]
+        public Task GetRemoteTaskTcp()
```

Comment:
You cannot do this. This leaks the existence of a database by the fact that you are using `DatabaseRequestHandler`
You need to use a `RequestHandler` and accept the database name externally.